src/schema/v2/artwork/index.ts

```diff
@@ -809,6 +806,12 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
           )
             return `Shipping: ${domesticShipping} ${shippingRegion} only`
 
+          if (
```

Comment:
So this whole `if` is redundant now as the cae would be covered above on line `767` and the code here would never be executed. So I think we should kill 809 - 814 lines here @lidimayra

Reply:
Absolutely, I was supposed to remove these, and I totally forgot about it! Just squashed this change